functions/jsBundle.js

```diff
@@ -7,10 +7,9 @@ const { jsBundle } = require("../src/jsBundle");
 
 const jsHandler = RavenLambdaWrapper.handler(Raven, async event => {
   try {
-    return jsBundle(event.queryStringParameters);
+    return await jsBundle(event.queryStringParameters);
```

Comment:
This await is required to make the Promise rejection from jsBundle be turned into a thrown error that we can catch.

Reply:
worth a comment in the file?

Reply:
Good idea, I've added one 👍